examples/example-gateway/idl/clients/baz/baz.thrift

```diff
@@ -45,4 +45,24 @@ service SecondService {
     string Echo(
         1: required string arg
     )
+
+    base.UUID EchoUUID(
+        1: required base.UUID arg
+    )
+
+    list<string> EchoList(
+        1: required list<string> arg
+    )
+
+    list<base.UUID> EchoUUIDList(
+        1: required list<base.UUID> arg
+    )
+
+    set<string> EchoSet(
+        1: required set<string> arg
+    )
+
+    map<base.UUID, base.BazResponse> EchoMap(
```

Comment:
Add a test for `map<>` with incomparable map key type?  like `map<base.BazRespoonse, bool>`?